Kitodo-DataManagement/src/main/java/org/kitodo/data/interfaces/WorkflowInterface.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.data.interfaces;
+
+import java.util.Objects;
+
+import org.apache.logging.log4j.util.Strings;
```

Comment:
Backport of comment from #6085 : 

A string comparing method from the log4j.util.Strings package is used here instead of using a similar method from the commons.lang3.StringUtils which is more common in use.

I'm writing this only on this place but should be changed on all places.

Reply:
This is actually always a mistake. Maybe isomeone one day feels compelled to create a checkstyle rule against this?